model/flow/protocol_state_test.go

```diff
@@ -126,7 +126,52 @@ func TestNewRichProtocolStateEntry(t *testing.T) {
 		assert.Equal(t, expectedIdentities, richEntry.NextEpochIdentityTable, "should be equal to next epoch setup participants + current epoch setup participants")
 	})
 
-	// TODO: include test for epoch setup phase where no prior epoch exist (i.e. first epoch setup phase after spork)
+	t.Run("setup-after-spork", func(t *testing.T) {
+		stateEntry := unittest.ProtocolStateFixture(unittest.WithNextEpochProtocolState(), func(entry *flow.RichProtocolStateEntry) {
+			// no previous epoch since we are in the first epoch
+			entry.PreviousEpochSetup = nil
+			entry.PreviousEpochCommit = nil
+			entry.PreviousEpoch = nil
+
+			// next epoch is setup but not committed
+			entry.NextEpochCommit = nil
+			entry.NextEpoch.CommitID = flow.ZeroID
+		})
+
```

Comment:
the following lines are sanity checks for the previously-constructed `stateEntry`, correct?  https://github.com/onflow/flow-go/blob/e22ad8b526931af2b5f8ddc6e2b2d98e90bf8bbe/model/flow/protocol_state_test.go#L141-L143
Would suggest to include a comment:

```suggestion
		// sanity check that previous epoch is not populated in `stateEntry`
```

Reply:
yeah, this is santity checks to ensure we are testing the correct thing